### PR TITLE
rgw: make test_multi.py work with python3

### PR DESCRIPTION
**Pull request by theanalyst**

### CODE REVIEW

**Comment by theanalyst** [comment]
tested locally, works for me on python2 and 3 right now

File: src/test/rgw/test_multi.py [hunk 1]
```diff
@@ -824,7 +829,7 @@ def init(parse_args):
         with file(path) as f:
```
Comment (BlaXpirit) [inline src/test/rgw/test_multi.py]: No `file` in Python 3

**Comment by BlaXpirit** [comment]
I actually have a pull request for Python 3 compatibility going. As far as I see, there is no critical difference between them other than one comment I made.

https://github.com/ceph/ceph/pull/10177/files#diff-11

However, my local [testing](https://gist.githubusercontent.com/BlaXpirit/6ce78ee907b2d13a9fce8de859173dc8/raw/2ccdb23959f68f12f0c1024bf428099679efff1d/test_multi.py.3) worked only with `PYTHONHASHSEED=0`, so it would be interesting to confirm that I missed something making the PR or testing went wrong.

**Comment by theanalyst** [comment]
@BlaXpirit yeah didn't see yours when I opened mine, I'll close this one after trying to find what is wrong :)

**Pull request closed without merging**